src/components/select/simple-select/simple-select.mdx

```diff
@@ -210,6 +210,26 @@ for users who rely on assistive technology.
   of={SimpleSelectStories.SelectWithDynamicallyAddedOption}
 />
 
+## Testing with RTL
+
+If you use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) in your project, you may need to mock the
+`getBoundingClientRect` method of the `Element` prototype in order to correctly test the `Select` component. You can do this manually in your
+test setup file, or import the `mockDOMRect` method to do it for you.
+
+```javascript
+import mockDOMRect from "carbon-react/lib/__spec_helper__/mock-dom-rect";
+
+describe("Select tests", () => {
+  beforeAll(() => {
+    mockDOMRect();
```

Comment:
suggestion (non-blocking): it might be worth demoing it with the type of values they'll need to pass

```suggestion
  mockDOMRect(200, 200, "select-list-scrollable-container");
```